docs/react_native_tutorials/react_native_1_setup_identifiers.md

```diff
@@ -52,11 +52,12 @@ Some libraries that we will be depending on make use of the node `crypto` packag
 browsers. These are not yet available to react-native/expo environments out of the box, so there is a bit of ceremony to
 bundle everything properly.
 
-Also, the metro bundler that react-native uses doesn't yet support the `cjs` file extension (
-see [facebook/metro#535](https://github.com/facebook/metro/issues/535)) which is used by some libraries in the stack, so
+Also, the metro bundler that react-native uses doesn't support the `cjs` file extension until v0.72.0 (see [facebook/metro#535](https://github.com/facebook/metro/issues/535)) which is used by some libraries in the stack, so
 we have to configure these too.
 
-#### `cjs` extension
+> ℹ️ **Note:** In case you run into issues check out the [Troubleshooting](../troubleshooting.md) page for some options.and example open-source React Native apps using veramo.
```

Comment:
```suggestion
> ℹ️ **Note:** In case you run into issues check out the [Troubleshooting](../troubleshooting.md) page for some options and example open-source React Native apps using Veramo.
```